routes/api.js

```diff
@@ -65,11 +65,15 @@ api.post('/friendships', async function (req, res) {
 })
 
 // Route for deleting user accounts and friendships
-api.delete('/users', function (req, res) {
-
+api.delete('/user/:secret/delete', function (req, res) {
```

Comment:
Change `/user/:secret/delete` to `/user/:secret` -- the DELETE method is already clear enough. We don't do GET `/user/:secret/get` for the same reason.